client/extensions/woocommerce/app/dashboard/manage-orders-view.js

```diff
@@ -84,45 +83,60 @@ class ManageOrdersView extends Component {
 		}
 	};
 
+	shouldShowPendingReviews = () => {
```

Comment:
Since this is live and likely won't be turned off, we could just remove this logic now that you are updating this component. Thought it could also be done in a follow-up PR that removes the config settings too.